client/src/components/Dropdown/Dropdown.js

```diff
@@ -44,13 +71,15 @@ class Dropdown extends Component {
     }
 
     return (
-      <div className={classnames('dropdown', className)}>
-        <div className="dropdown__button" onClick={this.handleClick}>
-          {button}
+      <div className={classNames('dropdown', className)}>
```

Comment:
After you change the `import` statement above, be sure to change this to `classnames`